### ISSUE TITLE
Incorrect speed when rendered to MIDI if the meter is 6/8

### ISSUE THREAD

**Issue by TCPN** [paulrosen/abcjs#124]
Hi, I found that if the meter is set to x/8 (e.g. 6/8, 12/8, 3/8) and the tempo set to 1/8=`SPEED` and then render the abc string as MIDI, the tempo of the rendered MIDI will actually be `SPEED * 1.5` eighth-notes per minute (i.e. 1/8=`SPEED * 1.5`).

You can check it with the example below.
```
X: 1
T: Test
M: 6/8
L: 1/8
Q: 1/8=100
K: A
CDECDE|CDECDB|CDECDE|DCBC3|]
```

**Comment by paulrosen** [comment]
I see the problem. I'll have a fix soon.

**Comment by paulrosen** [comment]
Will be fixed in v3.3.0, out soon.
